src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle() {
+    }
+
+    public RightTriangle(double firstLeg, double secondLeg) {
+        this.typeOfFigure = TypeOfFigure.RIGHT_TRIANGLE;
```

Comment:
You can pass random values to the constructor of a figure instead of generating them inside figure classes. Make sure that the values for `firstLeg` and `secondLeg` are being passed from outside rather than generated within the class.